UPDATING.md

```diff
@@ -62,6 +62,18 @@ More tips can be found in the guide:
 https://developers.google.com/style/inclusive-documentation
 
 -->
+
+
+### Made BaseSensorOperator respect the trigger_rule of downstream tasks
+
+Previously, BaseSensorOperator with setting soft_fail=True becomes skipped itself
+and skips all its downstream tasks unconditionally when it fails.
+The point is it does not respect the trigger_rule of downstream tasks when it fails.
+In the new behavior, the trigger_rule of downstream tasks are respected.
+User can preserve/achieve the original behaviour by setting every downstream task to all_success,
+because downstream tasks with trigger_rule all_success (i.e. the default) are skipped
+when upstream task is skipped.
+
```

Comment:
Sorry that I am bothering you again with that, but the UPDATING.md changed a lot i.e. we now have sections.
More information: https://github.com/apache/airflow/pull/10090

Please rebase and move that into an appropriate section.

After you've done that let me know and I am merging it. Sorry and thanks for your patience :)

Reply:
Thank you for sharing your information with me!
I've fixed! :)